packages/circuits/test/aadhaar-verifier.test.ts

```diff
@@ -79,6 +79,54 @@ function prepareTestData() {
       ),
   )
 
+  const tree = await newMemEmptyTrie();
+  const leafs = [
+    "4809579517396073186705705159186899409599314609122482090560534255195823961763", "2038038677412689124034084719683107814279606773706261227437666149072023632255",
```

Comment:
Can you please add comments here too? What are the first & second values on each line and how have you calculated them.